src/WrappedMToken.sol

```diff
@@ -662,13 +655,13 @@ contract WrappedMToken is IWrappedMToken, Migratable, ERC20Extended {
 
     /**
      * @dev    Wraps `amount` M from `account_` into wM for `recipient`.
-     * @param  account_   The account from which M is deposited.
      * @param  recipient_ The account receiving the minted wM.
      * @param  amount_    The amount of M deposited.
      */
-    function _wrap(address account_, address recipient_, uint240 amount_) internal {
+    function _wrap(address recipient_, uint240 amount_) internal {
```

Comment:
nit picking here: worth keeping 
 `_wrap(address account, address recipient, uint256 amount)`
 and `_unwrap(address account, uint256 amount)` the same as in `MExtension`